compiler/luci/service/src/Nodes/CircleQuantize.cpp

```diff
@@ -24,4 +27,10 @@ luci::CircleNode *CloneNodeLet<CN::OPQR>::visit(const luci::CircleQuantize *)
   return _graph->nodes()->create<luci::CircleQuantize>();
 }
 
+loco::TensorShape sinf::Algorithm::visit(const luci::CircleQuantize *node)
+{
+  const auto input_shape = luci::shape_get(node->input()).as<loco::TensorShape>();
```

Comment:
Could you use `circle_shape` function? `shape_get` is a temporary function, which will be removed in the future.

`CircleShapeInferenceHelper.h`
```cpp
// NOTE Functions in this namespace will be removed after new inference
//      algorithms are fully implemented.

// This function is temporary function for deprecating loco::shape_get
loco::NodeShape shape_get(const loco::Node *node);
```